.github/workflows/CompatCheck.yml

```diff
@@ -20,8 +20,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        downgrade_mode: ['alldeps']
-        julia-version: ['1.10', '1']
+        downgrade_mode: ['deps']
+        julia-version: ['1.10', '1.12']
```

Comment:
Is the fixation to 1.12 necessary for a specific reason? We will forget about this in the future I am afraid.